src/main/kotlin/com/grepp/nbe1_3_team04/stadium/api/request/validation/StadiumAllowedValues.kt

```diff
@@ -0,0 +1,14 @@
+package com.grepp.nbe1_3_team04.stadium.api.request.validation
+
+import jakarta.validation.Constraint
+import jakarta.validation.Payload
+import kotlin.reflect.KClass
+
+@Constraint(validatedBy = [StadiumAllowedValuesValidator::class])
+@Target(AnnotationTarget.VALUE_PARAMETER, AnnotationTarget.FIELD)
+@Retention(AnnotationRetention.RUNTIME)
+annotation class StadiumAllowedValues(
+    val message: String = "유효하지 않은 값입니다.",
+    val groups: Array<KClass<*>> = [],
+    val payload: Array<KClass<out Payload>> = []
```

Comment:
저랑 조금? 코드가 다른것 같은데 어떤 방식이 맞을지는 모르겠네요...

```suggestion
    val message: String = "유효하지 않은 값입니다.",
    val groups: Array<KClass<*>> = [],
    val payload: Array<KClass<out Any>> = []
```

저는 이러한 방식으로했습니다. Payload로 하면 타입이 안맞다고 하더라구요 아마도 저는 List를 써서 그럴까요..?

커스텀 어노테이션 제대로 공부하고 쓰는게 아니다보니까 이렇게 마이그레이션할때 조금 어려움이 있긴하네요~ 둘다 잘 작동하길 빌어봅니다